application/org.openjdk.jmc.flightrecorder.ui/src/main/java/org/openjdk/jmc/flightrecorder/ui/JfrEditor.java

```diff
@@ -123,7 +126,29 @@ public JfrEditor() {
 				}
 			}
 		};
+		if (FlightRecorderUI.getDefault().isWebsocketServerEnabled()) {
+			long websocketServerPort = FlightRecorderUI.getDefault().getWebsocketPort();
```

Comment:
why `long` for the variable type? it forces you to cast to int when calling `WebsocketServer` constructor below while `getWebsocketPort()` returns int anyway

Reply:
This is a leftover after refactoring, I was initially using IQuantity for the port because other numeric configs were using that, then I changed to long and finally to int. Good spot, fixed now!